### PR TITLE
detrend adjustment fixed for agg_data_model scoring failure

### PR DESCRIPTION
**Pull request by sayanchk**
This fixes the stationarity adjustment issue which was causing false positives

### CODE REVIEW

File: luminaire/model/window_density.py [hunk 1]
```diff
@@ -479,6 +479,7 @@ def _call_scoring(self, df=None, target_metric=None, anomaly_scores_gamma_alpha=
         :param luminaire.model.lad_structural.LADStructuralModel agg_data_model: Prediction model for aggregated data.
         :param str detection_method: Selects between "kldiv" or "sign_test" distance method.
         :param attributes: Model attributes.
+        :param agg_data: Aggregated Data per day.
```
Comment (shahsmit14) [inline luminaire/model/window_density.py:482]: Is it always going to be per day?

File: luminaire/model/window_density.py [hunk 2]
```diff
@@ -512,6 +515,7 @@ def _get_result(self, input_df=None, detrend_order=None, agg_data_model=None, va
         :param float anomaly_scores_gamma_alpha: Gamma fit alpha parameter.
         :param float anomaly_scores_gamma_loc: Gamma fit location parameter.
         :param float anomaly_scores_gamma_beta: Gamma fit beta parameter.
+        :param agg_data: Aggregated Data per day.
```
Comment (shahsmit14) [inline luminaire/model/window_density.py:518]: Same comment as before

File: luminaire/model/window_density.py [hunk 3]
```diff
@@ -704,6 +712,7 @@ def _anomalous_region_detection(self, input_df=None, window_length=None,
         :param luminaire.model.lad_structural.LADStructuralModel agg_data_model: Prediction model for aggregated data.
         :param luminaire.model.window_density.WindowDensityModel past_model: Past stored window density model.
         :param str detection_method: Selects between "kldiv" or "sign_test" distance method.
+        :param agg_data: Aggregated Data per day.
```
Comment (shahsmit14) [inline luminaire/model/window_density.py:715]: Same comment as before

File: luminaire/model/window_density.py (hunk 1 above)
Comment (sayanchk) [reply luminaire/model/window_density.py:482]: This is a higher level aggregated data which is per day at this moment

File: luminaire/model/window_density.py [hunk 4]
```diff
@@ -546,7 +550,8 @@ def _get_result(self, input_df=None, detrend_order=None, agg_data_model=None, va
                 data_adjust_forecast = 1
             adjusted_execution_data = execution_data / data_adjust_forecast
             if detrend_order > 0:
-                adjusted_execution_data = adjusted_execution_data.to_list()
+                data_adjust_forecast = agg_data[-1][-1]
```
Comment (shahsmit14) [inline luminaire/model/window_density.py:553]: Some logical code explanation would be helpful here

File: luminaire/model/window_density.py (hunk 2 above)
Comment (sayanchk) [reply luminaire/model/window_density.py:518]: This is a higher level aggregated data which is per day at this moment

File: luminaire/model/window_density.py (hunk 3 above)
Comment (sayanchk) [reply luminaire/model/window_density.py:715]: This is a higher level aggregated data which is per day at this moment

**Comment by shahsmit14** [comment]
If you are thinking to deploy this we would also need to update the version.

File: luminaire/model/window_density.py (hunk 1 above)
Comment (shahsmit14) [reply luminaire/model/window_density.py:482]: Got it, so the the comments are correct.

**Comment by sayanchk** [comment]
> If you are thinking to deploy this we would also need to update the version.

Yes. I just wanted to make sure changes are okay before that. Let me push that change

**Review by shahsmit14 — APPROVED** [review]

File: luminaire/model/window_density.py (hunk 4 above)
Comment (sayanchk) [reply luminaire/model/window_density.py]: added some comments

**Pull request merged**